{{cookiecutter.repo_slug}}/.github/workflows/stale.yml

```diff
@@ -21,20 +21,25 @@ jobs:
           stale-issue-label: "stale"
           exempt-issue-labels: "no-stale,help-wanted"
           stale-issue-message: >
-            There hasn't been any activity on this issue recently, so we
-            clean up some of the older and inactive issues.
+            There hasn't been any activity on this issue recently, and in order
+            to prioritize active issues, it will be marked as stale.
 
             Please make sure to update to the latest version and
             check if that solves the issue. Let us know if that works for you
-            by leaving a comment 👍.
+            by leaving a 👍
 
-            This issue has now been marked as stale and will be closed if no
-            further activity occurs. Thanks!
+            Because this issue is marked as stale, it will be closed and locked
+            in 7 days if no further activity occurs.
+
+            Thank you for your contributions!
```

Comment:
Not everyone thinks of "closed" as "locked to further comments," so making that clear.